app/models/ems_cluster.rb

```diff
@@ -63,6 +63,7 @@ class EmsCluster < ApplicationRecord
   # Provider Object methods - to be overwritten by Provider authors
   #
   supports_not :upgrade_cluster
+  supports_not :smartstate_analysis
```

Comment:
Hm this is a bit odd because no cluster supports smartstate_analysis, it is a question that shouldn't even be asked.  Taken to its extreme we would need to add this to every model

Reply:
EmsCluster actually has a `scan` defined.  So, it actually *does* support smartstate.

Reply:
:eyes: that's totally new to me haha, looks like it just saves the current record as a hash to drift_states.  Not sure I'd call that smartstate but okay